backend/package.json

```diff
@@ -28,8 +28,7 @@
     "neo4j-driver": "^4.2.3",
     "nest-neo4j": "^0.1.4",
     "reflect-metadata": "^0.1.13",
-    "rimraf": "^3.0.2",
-    "rxjs": "^6.6.6"
+    "rimraf": "^3.0.2"
```

Comment:
You have removed a dependency from the package.json but the yarn.lock is not committed